src/main/scala/io/scala/views/IndexView.scala

```diff
@@ -69,6 +69,7 @@ case object IndexView extends ReactiveView[IndexPage] {
   )
 
   lazy val tickets: Div = div(
+    htmlAttr("id", StringAsIsCodec) := "tickets", // no built-in "id" attribute??
```

Comment:
Yes there is, it should be `idAttr := "tickets"`

Reply:
J'ai passé 10 minutes à le chercher... fixup : dc2ee2c3d24d0c3b3edcdb25b62e548da823b45a